README.md

```diff
@@ -12,7 +12,16 @@ To include the layers within a yocto enviroment:
 
 1. add them to `conf/bblayers` file in your build directory
 
-1. build with a bitbake call e.g. `./bitbake iot-connect-image`
+1. add recipes to your image (in `local.conf` for example) - `IMAGE_INSTALL += " iot-c-sdk"`
```

Comment:
worthwhile mentioning that there's two ways to include the recipe outputs or packages. One is listed here, putting the above into your local build settings. Two, which I suspect is more likely, is having your own image recipe in your own layer,

So I suggest another line here saying something like:

"Add `IMAGE_INSTALL += " iot-c-sdk"` to meta-my-layer/recipes-core/images/my-image.bb"